sources/packages/web/src/components/generic/StatusInfoLabel.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <v-icon :icon="icon" :color="data.status" class="status-info-icon" />
```

Comment:
why `status-info-icon`??

Reply:
To match the icon size and properties as per figma.